### PR TITLE
Vignette updates

### LINKED ISSUE #160: Create tutorial vignette
Create introductory tutorial with simple example usage and explanation of inputs and outputs of `dgp_fun`, `method_fun`, `eval_fun`, `viz_fun`

### PR DESCRIPTION
**Pull request by tiffanymtang**
Update vignettes:

- Create short, introductory vignette
- Move current long vignette into a "comprehensive simChef guide"

Close #160

### CODE REVIEW

File: vignettes/figures/fit_eval_viz.png
Comment (jpdunc23) [inline vignettes/figures/fit_eval_viz.png:1]: Not critical, but any chance we can make this a pdf instead of png?

File: vignettes/cheat-sheet.Rmd
Comment (jpdunc23) [inline vignettes/cheat-sheet.Rmd:1]: Love it!

File: vignettes/simChef.Rmd
Comment (jpdunc23) [inline vignettes/simChef.Rmd:1]: Love this update. Way more approachable than the old version, I think it's going to be a huge help in getting people started with simChef.

File: vignettes/simChef-full.Rmd
Comment (jpdunc23) [inline vignettes/simChef-full.Rmd:1]: You've definitely made some big improvements to the flow and readability of the this vignette. 

Non-blocking for this PR, but do you think it would be possible to make this version build upon the `simChef.Rmd` rather than repeating what is already covered there? Then it would be a more advanced sequel rather than simply a more detailed version of the same, and hopefully we could cut it down a bit more. Something to think about but let's not worry about it for now.

**Review by jpdunc23 — APPROVED** [review]
Really huge improvements! Thanks so much for taking this on and doing such a lovely job.

I left some minor comments, none of which should block merging this PR. I haven't looked at the vignettes in the context of the pkgdown site, but we'll be able to see that once it's merged and can address any issues then, so feel free to merge when you have a moment.

Please remember to squash and merge.

File: vignettes/simChef-full.Rmd
Comment (tiffanymtang) [reply vignettes/simChef-full.Rmd:1]: Yeah, I think that would be good for the future. I was also thinking about whether or not to do this. I figured it could be beneficial to have another example for now. In the long-run, I think having (a) a separate collection of simChef examples plus (b) a comprehensive guide that builds on the introductory version would be ideal.

**Pull request merged**
